package.json

```diff
@@ -125,19 +131,10 @@
 		"typescript": "^5.2.2",
 		"unified": "^11.0.3",
 		"unist-util-visit": "^5.0.0",
+		"unocss": "^0.55.7",
 		"vite": "^4.4.9"
 	},
 	"svelte": "./dist/index.js",
 	"types": "./dist/index.d.ts",
-	"type": "module",
-	"dependencies": {
-		"@melt-ui/svelte": "^0.47.1",
-		"@unocss/preset-mini": "^0.55.7",
-		"@unocss/preset-wind": "^0.55.7",
-		"@unocss/reset": "^0.55.7",
-		"@unocss/transformer-directives": "^0.55.7",
-		"@unocss/transformer-variant-group": "^0.55.7",
-		"esm-env": "^1.0.0",
-		"unocss": "^0.55.7"
```

Comment:
I think let's leave `@melt-ui/svelte`, `@unocss/preset-wind`, `@unocss/transformer-directives`, and `unocss` as dependencies for now. The others we can safely add to devDependencies. And `esm-env` I think we can remove entirely if I'm not mistaken. I think it's from the previous version when I wasn't using Melt yet, and I also can't find it being used anywhere else, so please remove it.

Reply:
Ok I'll change that .. np